index.html

```diff
@@ -2,13 +2,18 @@
 <html>
 <head>
     <title>My Page</title>
-    <script type="text/javascript" src="scripts.js"></script>
+    <script type="text/javascript" src="scripts.js">
```

Comment:
It looks like you now reference the script twice in your html?  It should only be in one spot.

Reply:
The script tag is still in your HEAD and BODY.

Reply:
There are still 2 script tags here...

Reply:
Fixed.

Reply:
Nice!